content/docs/atom-guide/setup-atom-projects.md

```diff
@@ -1,111 +1,66 @@
 
-Atom Sample Viewer
-: 
-    ```cmd    
-    git clone https://github.com/o3de/o3de-atom-sampleviewer.git C:\AtomSampleViewer
-    ```
+2. After you set up your project, check out the following pages to explore Atom's features via the O3DE Editor:
+   - [Create a Level](/docs/learning-guide/tutorials/environments/create-a-level/): An introduction to your first level. Whenever you create a level in your project, the default environment consists of many Atom components. 
+   - [Component Reference - Atom](/docs/user-guide/components/reference/#atom): The full list of Atom components provided by the Atom Gem. 
 
-## Update the local project repos
 
-The community is continuing to update the Atom project repositories. This step ensures your local copy has the latest updates from the Atom repositories. 
+## Atom Sample Viewer
 
-In a command line window, navigate to the directory of the Atom repository and pull the latest changes from `origin/main` into your branch. 
+Atom Sample Viewer is a good place to start if you're interested in evaluating Atom's features in a controlled environment. It's a standalone application, meaning you don't need to open it with O3DE Editor. Rather, it showcases samples that demonstrate each feature individually and with parameters you can tweak and configure. 
 
-Atom Test
-: 
-    ```cmd
-    cd C:\AtomTest
-    git pull origin main
-    ```
+Check out the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository in GitHub.
 
-Atom Sample Viewer
-: 
-    ```cmd    
-    cd C:\AtomSampleViewer
-    git pull origin main
-    ```
+![An example of Atom Sample Viewer, a standalone application. This image showcases the Shadow sample.](/images/atom-guide/atom-sample-viewer/shadow.png)
 
 
-## Generate CMake project
+### Setup
 
-Use CMake to generate a Visual Studio solution, which is used in the next step to build the Atom project. 
+1. In a command line window, clone the [`o3de-atom-sampleviewer`](https://github.com/o3de/o3de-atom-sampleviewer) repository onto your local machine.
 
-Atom Test
-: 
     ```cmd
-    cmake -B "C:\AtomTest\build" -S "C:\AtomTest" -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages
+    git clone https://github.com/o3de/o3de-atom-sampleviewer.git
     ```
 
-Atom Sample Viewer
-: 
+2. Update your local repository by pulling the the latest changes, if any.
+
     ```cmd    
-    cmake -B "C:\AtomSampleViewer\build" -S "C:\AtomSampleViewer" -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages
+    git pull origin main
     ```
 
-## Register project to O3DE
-*This step is for the Atom Test project only.* 
-
-Register the Atom Test project to add it to the list of known projects in the O3DE manifest, located at `<user-directory>/.o3de/o3de_manifest.json`. 
+3. Use CMake to generate a Visual Studio solution, which is used in the next step to build the Atom project. 
```

Comment:
```suggestion
1. Use CMake to generate a Visual Studio solution. In the next step, you build the Atom project using this solution.
```
For flow and to avoid passive voice ("is used"), revised the second half of this sentence and split it into its own sentence.